pwndbg/aglib/remote.py

```diff
@@ -12,43 +12,3 @@
 @pwndbg.lib.cache.cache_until("objfile", "start")
 def is_remote() -> bool:
     return pwndbg.dbg.selected_inferior().is_remote()
-
-
-def is_debug_probe() -> bool:
```

Comment:
What happened to this?

Reply:
Ugh I see that this is unused. Seems that we maybe did some bug removing it :/

Reply:
previously it was implemented here https://github.com/pwndbg/pwndbg/pull/1402
logic to prevent downloading? symbols from remote debug probe